go.mod

```diff
@@ -34,4 +34,5 @@ require (
 	golang.org/x/sys v0.0.0-20210420205809-ac73e9fd8988 // indirect
 	golang.org/x/text v0.3.4 // indirect
 	gopkg.in/yaml.v2 v2.4.0
```

Comment:
is v2 still needed then?

Reply:
I think a bunch of our dependencies pull it in.